lib/HueSensor.js

```diff
@@ -2005,6 +2043,16 @@ HueSensor.prototype.addEntry = function (changed) {
         }
       }
       break
+    case 'room2':
+      {
+        var key = this.type.key === 'airqualityppb' ? 'voc' : this.type.key
```

Comment:
Please use `let` instead of `var`.  Probably better to make it `const` with a nested `?:` operator.